File: Estudos - repositorio/Ferramentas de Tecnologia/Java/Desafios - guanabara/desafio_5.java
// Faça um programa que peça um numero e informe seu sucessor e antecessor

import java.util.Scanner;
public class desafio_5 {

    public static void main(String[] args) {
        int antecessor, sucessor;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um número: ");
        int num = scanner.nextInt();
        
        antecessor = num - 1;
        System.out.println("Seu ANTECESSOR é: " + antecessor);

        sucessor = num + 1;
        System.out.println("Seu SUCESSOR é: " + sucessor);

        scanner.close();
        
    }
}

File: Estudos - repositorio/Ferramentas de Tecnologia/C++/Desafios Guanabara/desafio_5.cpp
#include <iostream>
#include <windows.h>

using namespace std;

int main(){

    SetConsoleOutputCP(CP_UTF8);

    int num;

    cout << "Digite um número: ";
    cin >> num;

    int sucessor = num + 1;
    cout << "Sucessor: " << sucessor;

    int antecessor = num - 1;
    cout << "\nAntecessor: " << antecessor;

    return 0;
}